### PR TITLE
feature/admin-panel

### PR DESCRIPTION
**Pull request by Jalal999**

### CODE REVIEW

**Comment by Jalal999** [comment]
image

image


image

File: components/Sidebar/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            width: drawerWidth,
+            marginTop: '70px',
+            boxSizing: 'border-box',
+          },
+        }}
+        variant="permanent"
+        anchor="right"
+        >
+            <List>
+                <ListItem disablePadding>
```
Comment (VSlobolinskyi) [inline components/Sidebar/index.js]: Map?

File: components/Forms/OrderForm/index.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ const OrderForm = () => {
             const newOrder = { customer, email, total };
             console.log(newOrder)
             const baseUrl = process.env.BASE_URL
-            await axios.post(`${baseUrl}/api/orders`, newOrder);
```
Comment (VSlobolinskyi) [inline components/Forms/OrderForm/index.jsx:26]: Why have you moved away from using variable here?

Comment (Jalal999) [reply components/Forms/OrderForm/index.jsx:26]: to run it first in localhost, as baseUrl variable is declared to the link of my project's deployment

**Comment by Jose-Hash-Dev** [comment]
I would suggest to add an error message for admin in UI, if delete/update/create throw an error

**Comment by Jalal999** [comment]
image

image

**Review by ilze-dom — APPROVED** [review]

File: components/PanelProductTable/index.jsx [hunk 3]
```diff
@@ -1,25 +1,45 @@
...
+  const [showAddForm, setShowAddForm] = useState(false);
+  const [errorAlert, setErrorAlert] = useState(false);
+  const [errorMsg, setErrorMsg] = useState("")
+  const [successMsg, setSuccessMsg] = useState("")
+
 
   const handleDelete = async (id) => {
     try {
-      await axios.delete(`http://localhost:3000/api/products/${id}`);
+      await axios.delete(`http://localhost:3000/api/products/5`);
```
Comment (Jose-Hash-Dev) [inline components/PanelProductTable/index.jsx:21]: I guess it should be `${id}`

**Review by Jose-Hash-Dev — CHANGES_REQUESTED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

File: components/PanelProductTable/index.jsx (hunk 3 above)
Comment (Jalal999) [reply components/PanelProductTable/index.jsx:21]: yes, I just wrote /5 to show error message. I will change it in the last update.

**Pull request merged**